testdata/no-leader/state.json.golden

```diff
@@ -0,0 +1,85 @@
+{
```

Comment:
There are 4 key things within this output state.

1. All servers should be marked as unhealthy in the `Servers.<uuid>.Health.Healthy` field
2. The overall `Healthy` field should be `false`
3. The `FailureTolerance` needs to be 0.
4. The `Leader` field should be empty

These are all desired values given that we have no leader.